lib/session.js

```diff
@@ -140,12 +140,12 @@ function fullpath(session, path) {
         path = path.substring(1);
     }
     return '/api/v' + session.apiVersion + '/' + path;
-};
+}
 
 function authValue(session, credentials) {
     if(credentials) {
         session.accessToken = null;
-        return 'Basic ' + (new Buffer(credentials.username + ':' + credentials.password)).toString('base64');
+        return 'Basic ' + (Buffer.from(credentials.username + ':' + credentials.password)).toString('base64');
```

Comment:
Creating new Buffers produces a deprecated warning and indicates we should switch to Buffer.from.